fpna/Utils/ExtentionMethods.ipynb

```diff
@@ -40,14 +64,12 @@
             "cell_type": "code",
             "source": [
                 "public static double[] GetLinearValues(this double initialValue, int shift, int length, double[] values){",
-                "\n    var linear = new double[length];",
-                "\n    ",
-                "\n    for (var i = shift; i < length; i++)",
-                "\n        if (i == shift)",
-                "\n            linear[i] = initialValue;",
-                "\n        else",
-                "\n            linear[i] = linear[i-1] * values[i - shift];",
+                "\n",
+                "\n    var linear = new double[shift + length];",
                 "\n    ",
+                "\n    for (var i = 0; i < length; i++)",
+                "\n        linear[shift + i] =  initialValue + values.Take(i + 1).Aggregate((x, y) => x + y);",
```

Comment:
.Sum() instead of Aggregate

Reply:
I prefer to keep the "Aggregate" to be able to move to usage "EvaluateFunctionElementByElement" method later